### DISCUSSION TITLE
Is this the intended startrail behavior? (Text overlay)

### DISCUSSION THREAD

**Discussion by ayryq** [aaronwmorris/indi-allsky#1661] [Q&A] [answered]
Got my first star trail last night. Is this normal to have the changing text overlay overlapping itself? (This is a screenshot from the video)
![vlcsnap-2024-11-21-07h24m05s321](https://github.com/user-attachments/assets/9b83ef59-6bb9-47b7-96d5-b76c47405d15)

**Comment by aaronwmorris** [comment, marked as answer]
Yes, this is normal behavior.  The star trails algorithm does not know anything about text or even stars.  It just takes the maximum value from all images and compiles them together.
